listenbrainz/webserver/templates/user/music_services.html

```diff
@@ -0,0 +1,140 @@
+{% extends 'base.html' %}
+{% from 'macros.html' import service_permission_button %}
+{% block title %}External Music Services - ListenBrainz{% endblock %}
+
+{% block content %}
+    <div id="user-profile">
+        <h2 class="page-title">Connect with third-party music services</h2>
+
+        <div class="panel panel-default">
+            <div class="panel-heading">
+                <h3 class="panel-title">Spotify</h3>
+            </div>
+            <div class="panel-body">
+                <div class="row">
+                    <div class="col-md-12">
+                        Connect to your spotify account to read your listening history, play music on ListenBrainz (requires
+                        Spotify Premium) or both. We encourage users to choose both options for the best experience but you
+                        may also choose only one option:
+                    </div>
+                </div>
+                <br>
+                <div class="row">
```

Comment:
Here is the HTML markup I used which should in the end replace that 'row' div:

I hope it's not too much work to make it fit…

```html
<div class="music-service-selection">
    <div class="music-service-option">
        <input type="radio" id="control_01" name="select" value="1" />
        <label for="control_01">
            <div class="title">
            Activate both features
            <br />
            <small>(Recommended)</small>
            </div>
            <div class="details">
            We will record your listening history permanently and make
            it available for others to view and explore. Discover and
            play songs directly on ListenBrainz, and import/export your
            playlist to and from Spotify.
            <br />
            <i>
                Note: Full length track playback requires Spotify Premium
            </i>
            </div>
        </label>
    </div>
    <div class="music-service-option">
        <input type="radio" id="control_02" name="select" value="2" />
        <label for="control_02">
            <div class="title">Play music on ListenBrainz</div>
            <div class="details">
            Discover and play songs directly on ListenBrainz, and
            import/export your playlist to and from Spotify.
            <br />
            <i>
                Note: Full length track playback requires Spotify Premium
            </i>
            </div>
        </label>
    </div>
    <div class="music-service-option">
        <input type="radio" id="control_03" name="select" value="3" />
        <label for="control_03">
            <div class="title">Record listening history</div>
            <div class="details">
            We will record your listening history permanently and make
            it available for others to view and explore.
            </div>
        </label>
    </div>
    <div class="music-service-option disable">
        <input type="radio" id="control_04" name="select" value="0" />
        <label for="control_04">
            <div class="title">Disable </div>
            <div class="details">
            Spotify integration will be disabled. You won&apos;t be able
            to import your listens or listen to music on ListenBrainz
            using Spotify.
            </div>
        </label>
    </div>
</div>
```

Reply:
Thanks. Based on this, I have updated the implementation.